frontend/src/components/files/File.tsx

```diff
@@ -435,9 +442,13 @@ export const File = (): JSX.Element => {
 							version={fileSummary.version_num}
 						/>
 					</TabPanel>
-					<TabPanel value={selectedTabIndex} index={3}>
-						<Listeners fileId={fileId} datasetId={datasetId} />
-					</TabPanel>
+					{datasetRole.role !== undefined && datasetRole.role !== "viewer" ? (
```

Comment:
Same could we use the "AuthWrapper" component here to wrap around?

Reply:
changed here too